### ISSUE TITLE
tests are missing

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/simphony-kratos#10]
There are no unit tests to test the wrapper or to ensure the cuds objects used by simphony-kratos "pass the same tests that the native cuds high-level objects pass".

**Comment by nathanfranklin** [comment]
This issue is required for integration.

**Comment by roigcarlo** [comment]
Dear @nathanfranklin 

I'am about to merge #9  as soon travis turns green.

This leaves still the CUDS objects tests ( mesh ) left.

I see that other wrappers implement their own version of the CUDS objects. The trick is that our wrapper works directly with the simphony object ( mesh for instance ). The run function gathers all the information needed from the simphony object at the begining and places it after the execution.

Bascially my point is that the unittest are going to deliver exactly the same results as the ones form the common/cuds.

I just want to confirm that, even redundat this is strictly necessary for the integration process

**Comment by nathanfranklin** [comment]
> I see that other wrappers implement their own version of the CUDS objects. The trick is that our wrapper works directly with the simphony object ( mesh for instance ). The run function gathers all the information needed from the simphony object at the begining and places it after the execution.
> 
> Bascially my point is that the unittest are going to deliver exactly the same results as the ones form the common/cuds.

yeah, i see your point.  you can use the testing template (i.e. simphony.testing.abc_check_particles) to test them but i agree with you that testing of the mesh redundant at this point and its probably better to just omit the mesh tests.
